index.html

```diff
@@ -11,6 +11,13 @@
       crossorigin="anonymous"
       referrerpolicy="no-referrer"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital@1&display=swap"
+      rel="stylesheet"
+    />
+
```

Comment:
Don't import the font in index.html, instead import it in index.scss and then create a variable for the font like 

```css

@import("url")

:root{
    --gdsc-font-1: "Poppins", sans-serif;
}
```